### PR TITLE
Align ToolButton Label in RTL mode

### PR DESCRIPTION
**Pull request by adilhaddaoui**
As per #1183 this fixes the alignment of the ToolButton Label in RTL mode as mentioned per this screenshot:
Screen Shot 2020-04-03 at 1 24 27 PM

### CODE REVIEW

**Comment by lipis** [comment]
Screenshot 2020-04-03 at 3 56 54 PM

I can still see it broken (collaboration)

**Comment by adilhaddaoui** [comment]
@lipis if you refresh the page when you change the language it will work.

The issue of alignment is fixed, now there is another issue that appeared is that when you change the language, the currentLanguage doesn't change until you refresh, and I'm creating a separate issue for that.

**Comment by lipis** [comment]
We need your magic here: @j-f1 :)

**Comment by lipis** [comment]
And then it breaks when switching to LTR mode.. we should find a better solution without updating the margins.. same margins for both versions

File: src/components/ToolIcon.scss [hunk 1]
```diff
@@ -33,6 +33,10 @@
 
   & + .ToolIcon__label {
     margin-left: 0;
+    &.rtl {
+      margin-right: 0;
+      margin-left: 0.8em;
+    }
```
Comment (j-f1) [inline src/components/ToolIcon.scss:39]:
```suggestion
    margin-inline-start: 0;
```

Comment (j-f1) [reply src/components/ToolIcon.scss:39]: (we should audit the codebase for uses of `left` and `right` and make sure they’re either switched to the semantic ones like `start` and `end` or properly handled in rtl)

**Comment by j-f1** [comment]
For future reference, I put a `dir=rtl` attribute on the `<html>` element when an RTL language is selected, so in the scss you can do `:root[dir="rtl"] & { ... }` to change the styles in RTL.

**Comment by lipis** [comment]
Could you update this PR accordingly with the right use of start/end @j-f1?

**Comment by j-f1** [comment]
I think we can revert the JS changes.

**Comment by lipis** [comment]
Yes please!

**Pull request merged**